### ISSUE TITLE
[en_GB] National Insurance number test is inconsistent

### ISSUE THREAD

**Issue by pimjansen** [FakerPHP/Faker#97]
Something seems up with the national insurance stuff. Either we have flakey generation, or a flakey test?

_Originally posted by @GrahamCampbell in https://github.com/FakerPHP/Faker/pull/96#issuecomment-736847412_

**Comment by bramceulemans** [comment]
I found the cause, it's not the bank account numbers, it's the new test for `Faker\Provider\en_GB\Person::nino`:

```
1) Faker\Test\Provider\en_GB\PersonTest::testNationalInsuranceNumber
Failed asserting that true is false.
```

**Comment by bramceulemans** [comment]
Fixed in #98
